net/probe-cli/Makefile

```diff
@@ -0,0 +1,90 @@
+#
+# Copyright (C) 2020 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=probe-cli
+PKG_VERSION:=3.0.0-rc.3
+PKG_RELEASE:=1
+
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://codeload.github.com/ooni/probe-cli/tar.gz/v$(PKG_VERSION)?
+PKG_HASH:=a600d1d2efaaf7c16d711a2f2b97d7b5af0c67bcb4d404ee0a20ca53a5d1f085
+
+PKG_MAINTAINER:=Jan Pavlinec <jan.pavlinec@nic.cz>
+PKG_LICENSE:=BSD-3-Clause
+PKG_LICENSE_FILES:=LICENSE.md
+
+PKG_BUILD_DEPENDS:=golang/host measurement-kit
+PKG_BUILD_PARALLEL:=1
+PKG_USE_MIPS16:=0
+
+GO_PKG:=github.com/ooni/probe-cli
+
+include $(INCLUDE_DIR)/package.mk
+include $(TOPDIR)/feeds/packages/lang/golang/golang-package.mk
+
+define Package/probe-cli
+  SECTION:=net
+  CATEGORY:=Network
+  TITLE:=OONI probe-cli
+  URL:=https://ooni.org
+  DEPENDS:=\
+	$(GO_ARCH_DEPENDS) +libevent2-openssl \
+	+libevent2-core +libevent2-extra +libevent2-pthreads \
+	+libmaxminddb +libnghttp2 +libopenssl \
+	+zlib +libcurl +libstdcpp
+endef
+
+define Download/probe-engine
+  VERSION:=f6fa6da45c8c51fbff2fb9c90048934cad84a451
+  SUBDIR:=probe-engine
+  FILE:=probe-engine-$$(VERSION).tar.xz
+  URL:=https://github.com/ooni/probe-engine.git
+  MIRROR_HASH:=02a8ce858e2925639bfa0bfcb8fa4a15101d1a3729877a4a76b9a8bf15ebaafd
+  PROTO:=git
+endef
+$(eval $(call Download,probe-engine))
+
+define Prepare/probe-engine
+	$(eval $(Download/probe-engine))
+	xzcat $(DL_DIR)/$(FILE) | tar -C $(PKG_BUILD_DIR) $(TAR_OPTIONS)
+	$(call PatchDir,$(PKG_BUILD_DIR)/probe-engine,./patches-probe-engine)
+endef
+
+define Download/bolt
+  VERSION:=94750aa2185e6ee4217105064949acace0156564
+  SUBDIR:=bolt
+  FILE:=boltdb-$$(VERSION).tar.xz
+  URL:=https://github.com/Psiphon-Labs/bolt.git
+  MIRROR_HASH:=d60cb3488a62e6a876d4e645caaebd7e7133ca50e94cdeecd7017f74c7267c73
+  PROTO:=git
+endef
+$(eval $(call Download,bolt))
+
+define Prepare/bolt
+	$(eval $(Download/bolt))
+	xzcat $(DL_DIR)/$(FILE) | tar -C $(PKG_BUILD_DIR) $(TAR_OPTIONS)
+	$(call PatchDir,$(PKG_BUILD_DIR)/bolt,./patches-bolt)
+endef
```

Comment:
Psiphon-Labs/bolt#1 has been merged (along with another commit that fixes the module name). Perhaps it would be better to patch probe-engine and/or probe-cli to use this updated version (by patching go.mod/go.sum), rather than download and patch bolt here?

Reply:
Fixed by patching go.mod and go.sum

Reply:
Related PR which should fix this in future https://github.com/ooni/probe-engine/pull/313